pkgs/development/tools/bazelisk/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  name = "bazelisk-${version}";
+  version = "0.0.3";
+
+  src = fetchFromGitHub {
+    owner = "philwo";
+    repo = "bazelisk";
+    rev = "v${version}";
+    sha256 = "1rkpw9izpav3ysb9fpbdf0m1wqrs3vl87s9zjjmfsjm5dfhxss72";
+  };
+
+  modSha256 = "1f73j6ryidzi3kfy3rhsqx047vzwvzaqcsl7ykhg87rn2l2s7fdl";
+
+  meta = with stdenv.lib; {
+    description = "A user-friendly launcher for Bazel";
+    homepage = https://github.com/philwo/bazelisk;
+    license = licenses.asl20;
+    maintainers = with maintainers; [ elasticdog ];
+    platforms = platforms.unix;
```

Comment:
remove this `platforms`, `buildGoModule` set it to `platforms.all` which reading the bazelisk readme seems to be the case
```suggestion
```

Reply:
@marsam Thank you...I've pushed an updated commit with all of your suggestions. For my own future benefit, how did you notice this? Knowing that the Go stuff typically supports Cygwin, from reading the source of `buildGoModule`, or is there a way to dump the final metadata after building?

@kalbasit It looks like the historical change to automatically set the `homepage` from the `goPackagePath` (via https://github.com/NixOS/nixpkgs/commit/40702af9365a6e5de71f9af3c930575c9ed0d426) may have gotten accidentally removed when `buildGoPackage` was shuffled around in a0d835e95d3e99d57e14adc6116e2b498f32f323 and 28435e47b149623dd55b83aee3b016063d9d391b. Was that intentional?

Reply:
@kalbasit never mind...looks like I was reading the previous version of `buildGoPackage` before it was moved, and not `buildGoModule`.